documentation/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.md

```diff
@@ -0,0 +1,93 @@
+# 2023-04-24 Project Proposal: Additional search views
+
+**Author**: @fcoveram
+
+## Reviewers
+
+- [x] @obulat
+- [ ] @krysal
+
+## Project summary
+
+The Openverse team has decided to enhance the browsing experience by
+incorporating new content views for providers, tags, and creators. This feature
+will allow users to share collections detached of individual searches.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
+The project adds three additional views for the Nuxt frontend where users
+interact in the following ways.
+
+### Results by tag
+
+Users land on the tag results page by clicking on the tag in the single result
+page or by opening a tag results link (for instance,
+[www.openverse.org/tags/image/cat](https://www.openverse.org/tags/image/cat)).
+
+The results page shows the name of the tag and all the items of the selected
+media type that have this exact tag applied. The items are sorted by
+`ingested_on` field in descending order.
+
+For instance, the `cat` tag page will show only images that have the `cat` tag,
+not items tagged with `cats` .
+
+### Results by source
+
+Users land on the source results page by clicking on the source name in the
+single result page or by opening a source results link (for instance,
+[http://www.openverse.org/providers/wikimedia/image/](https://www.openverse.org/providers/wikimedia/image/))
+
+The results page shows the name of the source and all the items from this source
+in the selected media type. The items are sorted by `ingested_on` field in
+descending order.
```

Comment:
Fair point, this query will be for an author on a single provider since we don't have a way to cross the works of an author through different providers/sources at the moment 👍